.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: ruby
 rvm:
-  - 2.2.0
-install: gem install rubocop -v 0.44.1
+  - 2.2.2
+install:
```

Comment:
If these tests require the netuitive_ruby_api will that need to be installed during the Travis CI build for tests? Probably worth it to add a `Gemfile` so these dependencies can be installed locally with `bundle install`. The `Gemfile` would look like:
```
source 'https://rubygems.org' do
  gem 'rubocop'
  gem 'haml-lint'
  gem 'mocha'
  gem 'activesupport'
end
```

The entire install section for Travis CI can be removed after adding a `Gemfile`, Travis CI runs `bundle install` on all Ruby projects by default.